include/net/net_if.h

```diff
@@ -220,9 +220,6 @@ struct net_if_ipv6 {
 	/** Prefixes */
```

Comment:
Also better would be just to say "Then each net_if_ipv6 can be added to the timer handler via a slist" in commit "net/iface: Coalesce all RS timers through one"